data/tlg0007/tlg068/tlg0007.tlg068.perseus-eng4.xml

```diff
@@ -81,3076 +88,2490 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
    <text>
       <body>
          <div type="translation" xml:lang="eng" n="urn:cts:greekLit:tlg007.tlg068.perseus-eng4">
```

Comment:
be careful here: there is a zero missing in the file name tlg007 should be tlg0007.